### PR TITLE
WIP: Added horizontal progress bar in Coach Report

### PR DESCRIPTION
**Pull request by christinepham**

### CODE REVIEW

**Comment by christinepham** [comment]
Horizontal progress bar that sums up all the in progress, complete and struggling in the selected facility. The number on the bar represents the total of each category.
screenshot 2015-08-25 14 20 51

File: kalite/coachreports/static/js/coachreports/coach_reports/hbtemplates/landing.handlebars [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div id="coach_report_wrapper">
 	<div id="summary_mainview">
-		<h1 class="status_name">{{ status.facility_name }}{{#if status.group_name }}: {{ status.group_name }}{{/if}}{{#if data.learner_events }}<button id="show_tabular_report" class="btn btn-success pull-right">{{_ "Show Tabular Report" }}</button>{{/if}}</h1>
+		{{!-- <h1 class="status_name">{{ status.facility_name }}{{#if status.group_name }}: {{ status.group_name }}{{/if}}{{#if data.learner_events }}<button id="show_tabular_report" class="btn btn-success pull-right">{{_ "Show Tabular Report" }}</button>{{/if}}</h1> --}}
```
Comment (rtibbles) [inline kalite/coachreports/static/js/coachreports/coach_reports/hbtemplates/landing.handlebars]: Could you uncomment this?

**Comment by rtibbles** [comment]
One comment to address.

Could you push the latest changes with the fix above, and I will resolve the merge conflict.

File: kalite/coachreports/static/js/coachreports/coach_reports/hbtemplates/landing.handlebars [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <div id="coach_report_wrapper">
 	<div id="summary_mainview">
-		<h1 class="status_name">{{ status.facility_name }}{{#if status.group_name }}: {{ status.group_name }}{{/if}}{{#if data.learner_events }}<button id="show_tabular_report" class="btn btn-success pull-right">{{_ "Show Tabular Report" }}</button>{{/if}}</h1>
```
Comment (rtibbles) [inline kalite/coachreports/static/js/coachreports/coach_reports/hbtemplates/landing.handlebars]: This should be here:
```
<h1 class="status_name">{{ status.facility_name }}{{#if status.group_name }}: {{ status.group_name }}{{/if}}</h1>
```

File: kalite/coachreports/static/js/coachreports/coach_reports/hbtemplates/landing.handlebars [hunk 3]
```diff
@@ -30,6 +29,20 @@
...
+		        	<li class="barlegend"><div class="struggling"></div><span>{{_ "Struggling" }}</span></li>
+		            <li class="barlegend"><div class="complete"></div><span>{{_ "Completed" }}</span></li>
+		       		<li class="barlegend"><div class="partial"></div><span>{{_ "In Progress" }}</span></li>
+	        	</ul>
+        	</div>
+        	<h5 class="innerbar bartext">
+				<strong>{{_ "Current status of user exercises" }}</strong>
+			</h5>
+		</div>
+		<h1 class="status_name">{{#if status.group_name }}: {{ status.group_name }}{{/if}}{{#if data.learner_events }}<button id="show_tabular_report" class="btn btn-success">{{_ "Show Tabular Report" }}</button>{{/if}}</h1>
```
Comment (rtibbles) [inline kalite/coachreports/static/js/coachreports/coach_reports/hbtemplates/landing.handlebars]: This line should be:
```
{{#if data.learner_events }}<button id="show_tabular_report" class="btn btn-success">{{_ "Show Tabular Report" }}</button>{{/if}}
```

**Pull request merged**